recipes/picard/2.7.1/build.sh

```diff
@@ -1,14 +0,0 @@
-#!/bin/bash
-
-TGT="$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM"
-mkdir -p "$TGT"
-mkdir -p "${PREFIX}/bin"
```

Comment:
Yes, but the git diff looks like you are removing a recipe from the bioconda git repository.